### PR TITLE
Es6 refactor resolver factory

### PR DESCRIPTION
**Pull request by rachel-ftw**
used let a bunch for those 'or' statements...

### CODE REVIEW

File: lib/ResolverFactory.js [hunk 1]
```diff
@@ -1,97 +1,98 @@
...
+const ModuleAppendPlugin = require("./ModuleAppendPlugin");
+const UnsafeCachePlugin = require("./UnsafeCachePlugin");
 
 exports.createResolver = function(options) {
 
 	//// OPTIONS ////
 
 	// A list of directories to resolve modules from, can be absolute path or folder name
-	var modules = options.modules || ["node_modules"];
+	let modules = options.modules || ["node_modules"];
```
Comment (sokra) [inline lib/ResolverFactory.js]: I guess most of these `let`s could be `const`s

File: lib/ResolverFactory.js [hunk 2]
```diff
@@ -1,97 +1,98 @@
+"use strict";
```
Comment (sokra) [inline lib/ResolverFactory.js]: use strict after comment

File: lib/ResolverFactory.js [hunk 3]
```diff
@@ -1,97 +1,98 @@
...
 		return true;
 	};
 
 	// The file system which should be used
-	var fileSystem = options.fileSystem;
+	const fileSystem = options.fileSystem;
 
-	// Use only the sync variants of the file system calls
-	var useSyncFileSystemCalls = options.useSyncFileSystemCalls;
+	// Use only the sync constiants of the file system calls
```
Comment (sokra) [inline lib/ResolverFactory.js]: typo

**Comment by ooflorent** [comment]
Thank you for your PR but it appears that the file was previously converted to ES6. I'm closing the PR for this reason.

**Pull request closed without merging**